SourceGen/RuntimeData/Commodore/1541-RAM-use.sym65

```diff
@@ -0,0 +1,203 @@
+byteStartBlk    @ $0038      ;Constant 7, ID mark for start of data block
+byteStartBlkHdr @ $0039      ;Constant 8, ID mark for start of block header
+dataBufParity   @ $003a      ;Parity for data buffer
+motorFlag       @ $003d      ;Motor flag
+curDrive        @ $003e      ;Active drive (FF, if not active)
+ctrlBufNum      @ $003f      ;Buffer number for disk controller
+gcrConvByteCnt  @ $0040      ;Byte counter for GCR conversion
+workCnt         @ $0041      ;Number of next work in queue (0 - 5)
+destTrk         @ $0042      ;Destination track (to move R/W head to)
+secPerTrk       @ $0043      ;Number of sectors per track for formatting
+tmpArea         @ $0044      ;Temp. work area; Scratch pad
+tmpArea2        @ $0045      ;Work code temp. storage
+byteBlkID       @ $0047      ;Data block ID char, default 07.
+headMoveCnt     @ $0048      ;Counter for head movement
+headStepCnt     @ $004a      ;Step counter for head transport
+lastReadSec     @ $004c      ;Last read sector
+nextSec         @ $004d      ;Next sector
+gcrConvBufPtr   @ $004e 2    ;Pointer to buffer for GCR->BIN conversion
+gcrFormatFlag   @ $0050      ;Flag for GCR format (0=BIN, 1=GCR)
+curFormatTrk    @ $0051      ;Current track number for formatting FF
+gcrConvBytesBuf @ $0052 4    ;Storage for 4 BIN bytes for GCR coding
+gcrConvStore    @ $0056 5    ;Storage for 5 GCR bytes
+headAccSteps    @ $005e      ;Number of steps for head motor accel/decl
+headAccFactor   @ $005f      ;accelerating/decl. factor 04
+headMoveVec     @ $0062 2    ;Pointer to routine for head movement $fa05
+headStepsMin    @ $0064      ;Minimum number of steps C8
+nmiVec          @ $0065 2    ;Pointer to start of NMI routine ($eb2e)
+nmiInProc       @ $0067      ;Flag: NMI in process
+diskInitInProc  @ $0068      ;Flag for disk initialisation
+secDivStepSize  @ $0069      ;Step size for sector division $0a
+numReadAttempts @ $006a      ;Number of read attempts 5
+ptrStartJumpTbl @ $006b 2    ;Pointer: Start of Jump table for U commands $ffea
+ptrStartBitmap  @ $006d 2    ;Pointer: Start of bitmap $0400
+ptrMBcmds       @ $006f 2    ;Pointer to address for M & B commands
+tmpArea3        @ $0071 4    ;Temp work area
+indPtrVar       @ $0075 2    ;Indirect pointer variable ($0100)
+listenAddr      @ $0077      ;Listener address (Device number + $20)
+talkAddr        @ $0078      ;Talker address (Device number + $40)
+curListenFlag   @ $0079      ;Flag: Active listener
+curTalkFlag     @ $007a      ;Flag: Active talker
+atnRecevFlag    @ $007c      ;Flag for ATN from serial bus receiving
+atnActiveFlag   @ $007d      ;Flag for ATN on serial bus active
+lastPrg         @ $007e      ;Last handled program
+driveNum        @ $007f      ;Drive number (on 1541 always 00)
+curTrk          @ $0080      ;Current Track number
+curSec          @ $0081      ;Current Sector number
+chnlNum         @ $0082      ;Channel number (Logical index)
+2ndAddr         @ $0083      ;Secondary address
```

Comment:
Symbols can't start with a number (here and following line).

Reply:
Ah, got it.  I loaded each one to see if errors popped up, but I must not have been looking in the right place.  I'll make the fixes this evening.  Thanks!

Reply:
fixed